### PR TITLE
feat: add release 1.24.1

### PR DESCRIPTION
**Pull request by nutellinoit**
as per title

### CODE REVIEW

**Review by ralgozino — APPROVED** [review]
LGTM

File: Furyfile.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 versions:
-  networking: v1.10.0
-  monitoring: v2.0.1
-  logging: v3.0.1
-  ingress: v1.13.1
-  dr: v1.10.1
-  opa: v1.7.3
-  auth: v0.0.2
+  networking: v1.12.1
```
Comment (omissis) [inline Furyfile.yml:6]: perhaps it's worth to bump to 1.12.2?

Comment (nutellinoit) [reply Furyfile.yml]: done

**Review by ralgozino — APPROVED** [review]
LGTM 🚀

**Pull request merged**